src/targets/mobile/actions/rating.js

```diff
@@ -0,0 +1,28 @@
+/* global __APP_VERSION__ */
```

Comment:
This is not how the code for the mobile app is structured. 
It's either

```
- actions
  - feature1
  - feature2
- reducers
  - feature1
  - feature2
```

or

```
- feature1
  - redux
  - other files
- feature2
  - redux
  - other files
```

Currently the code related to the mobile app follows the first pattern. We can change this, but that's not what this PR is about.